### ISSUE TITLE
Visual Studio "Add Class" brings up .Net Core Client Side file types instead Server Side Class file

### ISSUE THREAD

**Issue by jeffpapp** [aspnet/Tooling#452]
If you right click a folder in Solution Explorer and choose Add -> Class..., it brings up the Add Item dialog with .Net Core Client-side selected and a type script file as the default option.  In the past this has always defaulted to a new Class .cs file.  This seemed to change with VS 2015 Update 2.  Is this something that will go back to the old behavior with the new RC2 tooling?  Is there a way to configure this for the old behavior now?

![image](https://cloud.githubusercontent.com/assets/1260960/14570354/1486dfda-0301-11e6-838f-63139eb0156a.png)

**Comment by sayedihashimi** [comment]
We've seen this a few weeks back and fixed it I think. 

@phenning or @BillHiebert can you confirm?

**Comment by musabsudan** [comment]
Yeaaaah, this is reaaaally annoying, I started to develop a big enterprise app and every time I add a new class or an interface I have to change it from typescript to .cs 
I created lot's of .ts files by mistake.

let's dream about the ideal scenario :D 
I should use add class with server side as default whenever I am working outside of wwwroot and get the client side as default whenever I am working on wwwroot.

Cheers!
